server/src/create-svg.es6

```diff
@@ -0,0 +1,89 @@
+const util = require('./util.es6');
+const jsdom = require('jsdom');
+const { JSDOM } = jsdom;
+const moment = require('moment');
+const fs = require('fs-extra');
+
+const createPermit = {};
+
+const addRules = (permit, frag) => {
+  let treeHeightStumpDiameter = '';
+  if (permit.christmasTreesForest.treeHeight > 0 && permit.christmasTreesForest.stumpDiameter > 0) {
+    treeHeightStumpDiameter += `TREE HEIGHT MUST BE ${permit.christmasTreesForest.treeHeight} FEET OR LESS`;
+    treeHeightStumpDiameter += ` WITH DIAMETER ${
+      permit.christmasTreesForest.stumpDiameter
+    } INCHES OR LESS AT THE STUMP`;
+  } else if (permit.christmasTreesForest.treeHeight > 0) {
+    treeHeightStumpDiameter += `TREE HEIGHT MUST BE ${permit.christmasTreesForest.treeHeight} FEET OR LESS`;
+  } else if (permit.christmasTreesForest.stumpDiameter > 0) {
+    treeHeightStumpDiameter += `TREE DIAMETER MUST BE ${
+      permit.christmasTreesForest.stumpDiameter
+    } INCHES OR LESS AT THE STUMP`;
+  }
+  frag.querySelector('#tree-height-stump-diameter').textContent = treeHeightStumpDiameter;
+  if (permit.christmasTreesForest.stumpHeight > 0) {
+    frag.querySelector('#stump-height').textContent = `YOU MUST LEAVE A STUMP OF ${
+      permit.christmasTreesForest.stumpHeight
+    } INCHES OR LESS`;
+  }
+};
+
+
```

Comment:
there seems to be inconsistent whitespace between methods in this file.

Reply:
Updated